### PR TITLE
LLVM fix

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ before_script:
  - git submodule update --init rsx_program_decompiler asmjit 3rdparty/ffmpeg 3rdparty/pugixml 3rdparty/GSL 3rdparty/libpng Utilities/yaml-cpp 3rdparty/cereal
  - mkdir build
  - cd build
- - if [ "$TRAVIS_OS_NAME" = "linux" ]; then cmake ..; else cmake .. -DLLVM_DIR=/usr/local/opt/llvm38/lib/llvm-3.8/share/llvm/cmake; fi
+ - if [ "$TRAVIS_OS_NAME" = "linux" ]; then cmake .. -DLLVM_DIR=/usr/share/llvm-3.8/cmake; else cmake .. -DLLVM_DIR=/usr/local/opt/llvm38/lib/llvm-3.8/share/llvm/cmake; fi
```
Comment (danilaml) [inline .travis.yml]: You shouldn't need to specify LLVM location for linux, `find_package should `find it automatically (in the mac OS case it was required to use propper llvm and not apple-patched one).

**Comment by zerr10** [comment]
LLVM fix (skylake)

lol no work on i7 6700 3.4Ghz rx 460 4gb sahpire

**Comment by zerr10** [comment]
black scren all time.... and long load lol*

**Comment by zerr10** [comment]
the need for a new library or something? if so, please print them

**Comment by al0xf** [comment]
Which means the games start and that LLVM is working. Please read the FAQ on the forums, rpcs3 does only run a few games.

**Comment by zerr10** [comment]
but...
naruto work other option.. and dragon ball burst limit too why for now black ?

**Comment by al0xf** [comment]
The LLVM recompiler is unfinished.This commit fixes it crashing instantly. As you say you get a black screen now which is an improvement. No more questions on github please.

**Comment by zerr10** [comment]
ok ty

**Comment by AniLeo** [comment]
Please stop spamming.

**Comment by Ronaldo-brazil** [comment]
Is working well for me with i5 6400 with psn games running at 60 fps with api opengl and directx 12 already Vulkan the fps goes from 120 to 200 thanks nekotekina
capturar
capturar1
capturar2

**Comment by ikki84** [comment]
@zerr10 you will find some info in https://github.com/RPCS3/rpcs3/issues/1870 "PPU: LLVM recompiler issue #1870"

**Pull request merged**